src/Compilers/CSharp/Test/Syntax/Parsing/ParsingErrorRecoveryTests.cs

```diff
@@ -461,10 +463,12 @@ public void TestAttributeWithGarbageAfterName()
             Assert.NotNull(file);
             Assert.Equal(text, file.ToFullString());
             Assert.Equal(1, file.Members.Count);
-            Assert.Equal(SyntaxKind.IncompleteMember, file.Members[0].Kind());
-            Assert.Equal(2, file.Errors().Length);
-            Assert.Equal((int)ErrorCode.ERR_UnexpectedCharacter, file.Errors()[0].Code);
-            Assert.Equal((int)ErrorCode.ERR_SyntaxError, file.Errors()[1].Code);
+            Assert.Equal(SyntaxKind.FieldDeclaration, file.Members[0].Kind());
+            Assert.Equal(4, file.Errors().Length);
+            Assert.Equal((int)ErrorCode.ERR_NamespaceUnexpected, file.Errors()[0].Code);
```

Comment:
>Assert.Equal((int)ErrorCode.ERR_NamespaceUnexpected, file.Errors()[0].Code); [](http://example.com/codeflow?start=12&length=76)

Please switch to friendlier way to verify diagnostics.

Reply:
Done